compiler/daml-lf-proto/src/DA/Daml/LF/Proto3/DecodeV1.hs

```diff
@@ -420,7 +434,16 @@ decodeBuiltinFunction = pure . \case
   LF1.BuiltinFunctionFOLDR          -> BEFoldr
   LF1.BuiltinFunctionEQUAL_LIST     -> BEEqualList
   LF1.BuiltinFunctionAPPEND_TEXT    -> BEAppendText
+
   LF1.BuiltinFunctionERROR          -> BEError
+  LF1.BuiltinFunctionTHROW -> undefined -- TODO #8020
```

Comment:
Please use the full url of the issue. I grep it when releasing a new version of LF.
# FIXME: https://github.com/digital-asset/daml/issues/8020

Reply:
I'll add the full URL instances that will only be relevant when the version number changes, for example https://github.com/digital-asset/daml/pull/8114/commits/bfff8a08d53d08e94f3f717d03a6003c9c98acb5

In the meantime, just grep for `#8020` if you want to find the _temporary_ placeholders.